File: 1012 - Area/area.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){
    double a, b, c;
    cin >> a >> b >> c;
    printf("TRIANGULO: %.3f\n", (a * c) / 2.0);
    printf("CIRCULO: %.3f\n", acos(-1.0) * b * b);
    printf("TRAPEZIO: %.3f\n", ((a + b) * c) / 2.0);
}

File: 1012 - Area/area.py
import sys

a, b, c = map(float, sys.stdin.read().split())
print("TRIANGULO: {:.3f}".format((a * c) / 2.0))
print("CIRCULO: {:.3f}".format(3.14159 * c * c))
print("TRAPEZIO: {:.3f}".format(((a + b) * c) / 2.0))
print("QUADRADO: {:.3f}".format(b * b))
print("RETANGULO: {:.3f}".format(a * b))
